content/en/docs/tasks/observability/distributed-tracing/sampling/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Trace Sampling
+description: Learn the different approaches on how to configure trace sampling on the proxies.
+weight: 10
+keywords: [sampling,telemetry,tracing,opentelemetry]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
```

Comment:
We should be testing this. Any reason testing is not enabled?

Reply:
I wasn't sure if I should add tests or not. The only thing they would achieve is that "Istio starts", but we can't verify any sampling "worked" in the tests here. 

I mainly based this page on the similar one https://istio.io/latest/docs/tasks/observability/distributed-tracing/mesh-and-proxy-config/ (which we wanted to point to this one from)